src/code/bank1.asm

```diff
@@ -1388,18 +1388,22 @@ jr_001_5DCC::
     jr   nz, jr_001_5DCC                          ; $5DE3: $20 $E7
     ret                                           ; $5DE5: $C9
 
-func_001_5DE6::
+SaveFile::
```

Comment:
Maybe `SaveGameToFileˋ, to avoid a potential confusion with the actual save file ?